app/views/components/input_autocomplete.scala.html

```diff
@@ -46,8 +46,11 @@
     <div class="form-control-wrapper autocomplete__wrapper">
         <select aria-describedby="hint-@field.id" id="@field.id" name="@field.id" class="autocomplete">
             <option value="" selected disabled></option>
-            @for(option <- optionElements) {
-                <option id="@field.id-@{option.replace(" ","-")}" value="@option" @if(field.value.contains(option)){ selected }>@option</option>
+            @for(professionalBody <- optionElements) {
+                <option id="@field.id-@{professionalBody.name.replace(" ","-")}"
```

Comment:
Could this not produce some strange results e.g. from one I have found

This - > Royal College of Nursing (J - Nursing Standard)
Would look like this -> Royal-College-of-Nursing-(J---Nursing Standard)